src/tests/ftest/erasurecode/rebuild_disabled_single.yaml

```diff
@@ -7,7 +7,7 @@ hosts:
     10_server:
       test_servers: server-[1-5]
   test_clients: 1
-timeout: 400
+timeout: 1900
```

Comment:
This is quite a jump.  Out of the 5 repeated runs 1561.57s is the longest runtime I observed in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13848/25/artifact/Functional%20Hardware%20Large/erasurecode/rebuild_disabled_single.py/repeat002/results.html.  Maybe we could reduce by a couple hundred seconds?  The other thing we might/probably should do for this test - or it would be a very good experiment - would be to add a `crt_timeout: 10` entry for the server config to see if that helps lower the runtime of this test.

Reply:
I had 5 repeat test loop, with "crt_timeout: 10",  the longest test-time is 541 seconds.  I am reduce the test-time to 700, please see the new commit.